app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -50,7 +50,7 @@
     private BluetoothAdapter bluetoothAdapter;
     private IBluetoothHandler bluetoothHandler;
     private Set<BluetoothDevice> pairedDevices;
-    private boolean isMaster = false;
+    private boolean isMaster;
```

Comment:
Hmm, it seems that Intellij has no option to give a warning when you forget to initialize a field. In my opinion it's unimportant if we do or do not initialize fields to their default values. Consistency is more important I think and because Intellij can only (I agree that that should not be the case) give a warning if you have a redundant initializer, we can only enforce consistent code if we do not initialize fields to their default value.